routes/middlewares/festival/lol/post/postLolVote.ts

```diff
@@ -0,0 +1,51 @@
+import CustomError from '@Middleware/error/customError';
+import { NextFunction, Request, Response } from 'express';
+
+import LolTeam from '@Model/lolTeam.model';
+import LolVote from '@Model/lolVote.model';
+
+const postLolVote = async (req: Request, res: Response, next: NextFunction) => {
+  const team_pk: number = req.body.team_pk;
+  const user_pk: string = res.locals.user.pk;
```

Comment:
```suggestion
  const user: User = res.locals.user;
```
다른 파일 보면 알겠지만 저렇게 불러오지 않음. 타입스크립트를 쓰는 이유가 없어짐